src/core_modules/capture-core/components/Pagination/withDefaultNavigation.js

```diff
@@ -19,9 +19,8 @@ const styles = (theme: Theme) => ({
 });
 
 type Props = {
-    rowsPerPage: number,
+    isPaginationButtonDisabled: boolean,
```

Comment:
NIT: could we go with a more precise name here?

Reply:
you mean such as `isPaginationNextButtonDisabled`?

Reply:
Yes. Suggestion: `nextPageButtonDisabled`

Reply:
You mind if wit is `isNextPageButtonDisabled `

Reply:
`isNextPageButtonDisabled` sounds more like a function to me...

Reply:
Fair 👌  There is a convention by some in which, `is` at the beginning of a variable name indicates boolean.

I will go with your suggestion to make it clear for both of us